SmartCarApplication/app/src/main/java/com/example/smartcarapplication/MainActivity.java

```diff
@@ -81,8 +84,6 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
                 return true;
             }
         });
-        //mCameraView = findViewById(R.id.imageView);
```

Comment:
I think we need this for the cam feed to be broadcasted on the app 🎥, but you did not have it in your branch made, so can't blame, no worries.